### PR TITLE
Feature/a2 604 file upload 2

### PR DESCRIPTION
**Pull request by HadrianHughesKC**
## Describe your changes

* refactor(api): change document service methods to only require appeal ID
* feat(api): create an endpoint to return folders by appeal ID and path
* feat(web): fetch attachments folder ID on IP comment upload page
* feat(web): pass real values for blob storage from config
* feat(web): set the nextPageUrl for the manual IP comment upload page
* feat(web): pass documentStage and documentType arguments to file upload
* fix(web): ensure useBlobEmulator config variable is a boolean

Will need to test file upload in dev.

## Issue ticket number and link

[A2-604](https://pins-ds.atlassian.net/browse/A2-604)

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

[A2-604]: https://pins-ds.atlassian.net/browse/A2-604?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: appeals/api/src/server/endpoints/documents/documents.controller.js [hunk 1]
```diff
@@ -32,11 +32,29 @@ import { EventType } from '@pins/event-client';
...
+ * @param {Request} req
+ * @param {Response} res
+ * @returns {Promise<Response>}
+ * */
+const getFolders = async (req, res) => {
+	const { appeal } = req;
+	const { path } = req.query;
+
+	if (path) {
+		const folders = await service.getFolderByPath(appeal.id, /** @type {string} */ (path));
```
Comment (Si3rr4wow) [inline appeals/api/src/server/endpoints/documents/documents.controller.js:50]: man I miss `as string`

File: appeals/api/src/server/openapi-types.ts [hunk 2]
```diff
@@ -119,7 +119,7 @@ export interface ValidateDate {
 
 export interface AppellantCaseData {
 	casedata?: {
-		/** @example "ba35248c-0b50-4352-ac3e-89d5c8fd165f" */
+		/** @example "40df3b3d-fc9c-4e65-85db-4a9c383e0fcf" */
```
Comment (Si3rr4wow) [inline appeals/api/src/server/openapi-types.ts:122]: Rare. Not that this isn't ok but why update this example? 😅

**Review by Si3rr4wow — APPROVED** [review]
🚀

File: appeals/api/src/server/openapi-types.ts (hunk 2 above)
Comment (HadrianHughesKC) [reply appeals/api/src/server/openapi-types.ts:122]: It was part of the change when I regenerated the file, not sure why though :)

**Pull request merged**